scripts/build.ps1

```diff
@@ -700,7 +700,10 @@ function Update-VsixVersion($vsixProjectDir)
     Write-Log "Update-VsixVersion: Started."
 
     $packageDir = Get-FullCLRPackageDirectory
-    $vsixVersion = $Version
+
+    # The last insertion for rel/15.6 had 15.7.0-* version
```

Comment:
Cherry-pick 3ce38d292805c842611e54ca42b8cd0ad74a0d1d and remove VSTestVersionPrefix variable from build defination.